docs/Using-Fleet/MDM-commands.md

```diff
@@ -71,5 +71,18 @@ Fleet UI:
 
 2. Look at the on-screen information.
 
+## List of recent commands
+
+You can view the list of recently executed commands using "fleetctl":
+
+1. Run `fleetctl get mdm-commands`
+2. View the list of recently executed commands, most recent first, along with the timestamp, targeted hostname, command type, execution status and command ID.
+
+The command ID can be used to view command results as documented in [Step 4 of the previous section](#step-4-view-the-commands-results). The possible status values are:
+* Pending: the command has yet to be executed on the host.
+* Acknowledged: the command was executed successfully.
```

Comment:
```suggestion
* Acknowledged: the host responded with "Acknowledged" status via the MDM protocol: the host processed the command successfully.
```

I think we should be more explicit here about how "Acknowledged" is a status that comes back from the MDM protocol. A lot of users express problems of getting Acknowledged back and not really knowing if the command ran or not.

I also borrowed Apple's [definition of Acknowledged](https://developer.apple.com/documentation/devicemanagement/installprofileresponse): 

![Screenshot 2023-05-30 at 4 21 39 PM](https://github.com/fleetdm/fleet/assets/47070608/2f75e4ca-78dc-46f7-93dc-753ca6157c2d)